guides/Retrospectives.md

```diff
@@ -0,0 +1,70 @@
+# Retrospectives
+
+## Why run retrospectives?
+Despite the size of a team or scope of a project there is always room for improvement. Running a retro allows the team to iterate on the existing processes and improve the way the team works. It is also a good opportunity to check in with other team members to collaboratively assess team health. In short, the retrospective helps the team identify how they can improve their processes and most efficiently achieve business goals.
+
+## What is a retrospective?
+A retrospective is a meeting held typically at the end of a sprint that gives the team the opportunity to reflect on what happened in the sprint and identify actions for improvement moving forward.
+
+## When to run retrospectives?
+Generally, a retrospective is held at the end of a sprint. Holding it at the end of a sprint is suggested because the team has completed a sizeable amount of work that they can reflect on. It also allows the team to quickly implement ideas and improvements in the next sprint (that can then be followed up on and discussed in the next retro). The most important thing regarding when retros are held is to have them frequently enough to allow for iteration and continuous improvement.
+
+## Who should join a retrospective?
+Every team member (such as a PM, a designer, developers, and an analyst when necessary) is encouraged to join a retrospective. Also, it may be valuable to include other stakeholders when a notable milestone is completed. For example, after the launch of the Year In Art project it might be helpful to include an editorial team member for that retrospective. It is also important to be mindful of the meeting size so as to keep the focus of that retrospective manageable.
+
+## How to run retrospectives?
+
+__Facilitator__
+A retro needs a facilitator whose role is to:
+
+- Manage time.
+- Assist the group in ordering/organizing items.
+- Engage and encourage discussion (by asking questions/reviewing items).
+- Take notes of action items.
+
+
+Many articles, blog posts, and books suggest having a dedicated facilitator rather than having one of the team members play this role, which would help remove bias. In a small team, that may not be an option but is something for the facilitator to keep in mind.
```

Comment:
Plus one; I've also found success (for teams that are working together consistently) in having the facilitator rotate every sprint; helps builds empathy in leading/participation and distribute sense of ownership throughout the process. Also can be an opportunity to help build facilitation skills for more people! 

Other added benefit, does not burden an individual to run the retro each time.